### PR TITLE
Added PHP Yorkshire elephpant

### PR DESCRIPTION
**Pull request by carnage**
putting this in as a placeholder; will update with photos when I have some

### CODE REVIEW

**Comment by philipsharp** [comment]
What is the status of this elephpant? Is there a prototype or a campaign?

File: source/_elephpants/2018-04-14-php-yorkshire.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+categories:
+    - PHP Yorkshire
```
Comment (philipsharp) [inline source/_elephpants/2018-04-14-php-yorkshire.md]: This should be a lower-cased, non-spaced string like `phpyorkshire`. It's used as an array key.

**Comment by philipsharp** [comment]
Great! In addition to the one change I noted, you also need to define a common and subspecies name here: https://github.com/carnage/afieldguidetoelephpants/blob/26e4622a953519c69b3e583ca94f9b00d1b5aa5c/app/config/sculpin_site.yml#L66

A little Googling suggests _"Eboraci"_ as an appropriate subspecies, but it's up to you.

**Comment by carnage** [comment]
OK, this should be ready to go now

**Comment by philipsharp** [comment]
Thanks!

**Comment by philipsharp** [comment]
Is "Jorvik" a nickname for the user group, or the elephpant?

**Pull request merged**